extensions/scheduler/src/helper/windows/schtasks-xml-generator.ts

```diff
@@ -153,58 +159,35 @@ ${actions}
     const weekdays = expandField(components.weekday, 7);
 
     // Generate triggers for each time combination
-    const triggers: string[] = [];
+    const calendarTriggers: CalendarTrigger[] = [];
 
     const hoursList = hours.length ? hours : [0];
     const minutesList = minutes.length ? minutes : [0];
 
     for (const h of hoursList) {
       for (const m of minutesList) {
         const time = `${String(h).padStart(2, '0')}:${String(m).padStart(2, '0')}:00`;
+        const startBoundary = `2025-01-01T${time}`;
 
         if (weekdays.length > 0) {
           // Weekly trigger
           const daysOfWeek = weekdays.map(w => this.getDayOfWeekName(w)).join('');
-          triggers.push(`    <CalendarTrigger>
-      <StartBoundary>2025-01-01T${time}</StartBoundary>
-      <Enabled>true</Enabled>
-      <ScheduleByWeek>
-        <DaysOfWeek>
-          ${daysOfWeek}
-        </DaysOfWeek>
-        <WeeksInterval>1</WeeksInterval>
-      </ScheduleByWeek>
-    </CalendarTrigger>`);
+          calendarTriggers.push({ startBoundary, daysOfWeek });
         } else if (days.length > 0) {
           // Monthly trigger with specific days
-          const daysXml = days.map(d => `<Day>${d}</Day>`).join('\n          ');
-          triggers.push(`    <CalendarTrigger>
-      <StartBoundary>2025-01-01T${time}</StartBoundary>
-      <Enabled>true</Enabled>
-      <ScheduleByMonth>
-        <DaysOfMonth>
-          ${daysXml}
-        </DaysOfMonth>
-        <Months>
-          <January /><February /><March /><April /><May /><June />
-          <July /><August /><September /><October /><November /><December />
-        </Months>
-      </ScheduleByMonth>
-    </CalendarTrigger>`);
+          const daysOfMonth = days.map(d => `<Day>${d}</Day>`).join('\n          ');
```

Comment:
nitpick: there is a bit of xml left in this string interpolation, it could be moved to .mustache with the rest I think

Reply:
But I guess it is complicated so I will resolve that